ipatests/test_integration/test_sssd.py

```diff
@@ -50,6 +50,7 @@ def install(cls, mh):
         tasks.install_adtrust(cls.master)
         tasks.configure_dns_for_trust(cls.master, cls.ad)
         tasks.establish_trust_with_ad(cls.master, cls.ad.domain.name)
+        cls.ad_group = 'group@group@{0}'.format(cls.ad.domain.name)
```

Comment:
nitpick: this group is specific to a  single tests and is not likely to be used in any other test.
So there is no reason to store it in common setup, you can define it in the added test case